stack.yaml

```diff
@@ -17,7 +17,7 @@ flags: {}
 extra-package-dbs: []
 
 # Control whether we use the GHC we find on the path
-# system-ghc: true
+system-ghc: false
```

Comment:
Please don't commit this change. (It's fine to have it on your local machine, but does not belong in this pull request.)